CONTRIBUTING.md

```diff
@@ -121,6 +121,16 @@ this:
 You can format individual files in place by running `clang-format -i <file>...`
 from any directory within this project.
 
+There is a Continuous Integration job that runs clang-format on PRs. If the code does not comply, an artifact will be generated which is a patch file that corrects any auto-fixable formatting issues.
+
+To download the patch file:
+
+1. Next to `clang-format / check (pull_request) Failing after #s` -> click **Details** to open the details page.
```

Comment:
```suggestion
1. Click **Details** next to the `clang-format / check (pull_request) Failing after #s`.
```

Reply:
Reading the original text left-to-right, it tells you what to look for - in the order that you need to look for it. In other words, you need to look for `clang-format / check (pull_request) Failing after #s` before you can click **Details**. In the way that you propose, it's as if the steps are reversed.